design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -1,91 +1,110 @@
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import homepage_banner from '../static/images/homepage_banner.svg';
 
-export const Headline = styled.div`
+export const BannerBackground = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 550px;
+  background-image: url(${homepage_banner});
+  padding-left: 100px;
+  padding-right: 100px;
+  margin-bottom: 90px;
+`;
+
+export const BlurbContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  align-items: left;
+  justify-content: center;
+  text-align: left;
+  h3 {
+    height: 72px;
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: bolder;
+    font-size: 30px;
+    line-height: 47px;
+    padding-left: 50px;
+  }
+
+  p {
+    font-size: 20px;
+    line-height: 28px;
+    color: black;
+    padding-left: 50px;
+    font-weight: 500;
+  }
+`;
+
+export const Blurb = styled.div`
   text-align: center;
+  display: block;
+  justify-content: center;
+  position: absolute;
+  width: 497px;
+  height: 348px;
+  right: 120px;
+  top: 190px;
+  background: rgba(255, 255, 255, 0.85);
```

Comment:
Add color values to `constants/colors.ts`. Then in this file, call `import { colors } from '../constants/colors';` at the top and `${colors.colorName}` in place of the rgba.